hrjobcontract/js/src/job-contract/directives/number.js

```diff
@@ -9,8 +11,11 @@ define([
     return {
       require: 'ngModel',
       link: function ($scope, el, attrs, modelCtrl) {
-        var toFixedVal = 2,
-          notToFixed = attrs.hrjcNumberFloat || false;
+        var toFixedVal = 2;
+        var notToFixed = attrs.hrjcNumberFloat || false;
+        var toHoursRound = 0.25;
```

Comment:
I am not sure of this line `var toHoursRound = 0.25;`, could you explain a bit please.

Reply:
In Leave in Hours we allow to break hours to 15 minutes intervals. 1 hour * 0.25 = 15 minutes.

Reply:
Okay, could you please add this comment to the code also.